wordpress-mobile/gutenberg-mobile
# Release v1.22.0

Release for Gutenberg Mobile v1.22.0

## Related PRs

* Gutenberg: https://github.com/WordPress/gutenberg/pull/20062
* WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/11224
* WPAndroid after:https://github.com/wordpress-mobile/WordPress-Android/pull/11225
* WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/13353
* WPiOS after: https://github.com/wordpress-mobile/WordPress-iOS/pull/13354
* FluxC-Android: https://github.com/wordpress-mobile/WordPress-Android/pull/11227
* Aztec-Android: https://github.com/wordpress-mobile/AztecEditor-Android/pull/892
* Other repos: No changes in other repos

## Extra PRs that landed after the release was cut

* #1856 Fix gallery size labels
* #1867 Send selection change event when typing
* #1871 Enable Dismiss on PlainText in Android

## Changes

Gathered via `git log v1.21.0..release/1.22.0 --first-parent` and trimmed depending on whether the change is functional or not.

* #1830 (DEV only) Fix gallery upload sync
* #1806 Added list block ordered button e2e test
* #1686 Adding retry feature to mobile apiFetch handler
* #1847 Remove DEV flag from Media Editor button
* #1841 Show the media edit icon only if the block is selected
* #1817 External media links from another app will not work if the block editor is enabled
* #1835 Make inserter to show options on long-press to add before/after
* #1833 Project cleanup
* #1825 Fix: fix appender style
* #1826 Update ref to react-native-slider fork to support autolinking
* #1827 Fixes the Image's media button not displaying the Picker
* #1782 Add media edit icon to image block
* #1821 Fix: AppenderButton and Separator in InnerBlock
* #1813 Android e2e test flakiness fix
* #1776 (DEV only) Allow configuration of selection/caret color on Aztec view
* #1820 Rename iOS module name of sample app 
* #1450 Update React Native to 0.61.5
* #1666 Fix broken link in Getting Started documentation
* #1785 (DEV only) Style improvements for template previews
* #1799 Adjust to Gutenberg master to fix issues with icons, the primitives package and a new dependency for data
* #1762 (DEV only) Use patch-package to fix max-depth issue in react-native (temporary)
* #1786 Hardcode Android RN version in bridge and aztec

## To test:

* Use the main WP apps to test the changes above. See WPAndroid and WPiOS PRs if there are specific tests to run.
* Smoke test the main WP apps for general writing flow.

PR submission checklist:
